### ISSUE TITLE
The Initializer container Job is stuck and doesn't complete the Job because of the CloudSQL Proxy

### ISSUE THREAD

**Issue by bruno561** [DefectDojo/django-DefectDojo#7235]
**Be informative**
When using Cloud SQL as a managed database, the Job related to Initializer isn't completed because your Sidecar is still running. Sidecar is cloudsql-proxy.

**Bug description**
As described in the informational section, the Initializer Job fails when using CloudSQL. NotReady status is displayed.

Image of the logs of the container named Initializer. Logs stop at this point:
![image](https://user-images.githubusercontent.com/65369082/205642316-2a74f35e-f5f4-4770-8c0e-34b9d32ab0fc.png)

Image of Pod related to Job with status of NotReady.
![image](https://user-images.githubusercontent.com/65369082/205642600-35a3a7ef-b82b-4e37-abdc-2c0dbadb89ce.png)

**Steps to reproduce**
Basically, just use Cloud SQL in the Values file.
```
  # Google CloudSQL support in GKE via gce-proxy
  cloudsql:
    # To use CloudSQL in GKE set 'enable: true'
    enabled: true
```

**Expected behavior**
Have some way that when the initializer container finishes the job the cloudsql-proxy container is deleted, in order to finish the Job.

**Deployment method** *(select with an `X`)*
- [ ] Docker Compose
- [X] Kubernetes
- [ ] GoDojo

**Environment information**
- GKE (Google Kubernetes Engine) version 1.23.12-gke.100
- https://github.com/DefectDojo/django-DefectDojo/tree/2.16.2/helm/defectdojo

**Logs** 
![image](https://user-images.githubusercontent.com/65369082/205642316-2a74f35e-f5f4-4770-8c0e-34b9d32ab0fc.png)
The cloudsql-proxy container hangs at this point:
![image](https://user-images.githubusercontent.com/65369082/205643890-18100a29-56b9-4017-9473-df17b52dfb4a.png)

**Comment by tellierd** [comment]
Has anyone worked on a fix for this? I'm having the very same issue. Thx

**Comment by mtesauro** [comment]
@tellierd TBH, Helm isn't an [officially supported deployment method](https://github.com/DefectDojo/django-DefectDojo/issues/7235#issuecomment-1721522661) for DefectDojo so we rely on the community to submit PRs for the Helm chart in this repo.

There are far to many deployment choices for k8s/Helm for an open source project to able to test them all.

**Comment by jndeverteuil** [comment]
With PR #10824 the bug is now fixed from what I tested

Also, PR #11363 has fixed the helm charts not being published

This means, the bugfix is available in Helm Chart version 1.6.164

**Comment by mtesauro** [comment]
Given that we have a report that this is working, I'm going to go ahead and close this.

Reopen or create a new issue if that's not the case.
